deploy/crds/app_v1alpha1_kieapp_crd.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: kieapps.app.kiegroup.org
+spec:
+  group: app.kiegroup.org
+  names:
+    kind: KieApp
+    listKind: KieAppList
+    plural: kieapps
+    singular: kieapp
+  scope: Namespaced
+  version: v1alpha1
```

Comment:
i can go either way. one of the benefits of the newer sdk is ability to increment api versions and keep older versions for backwards compatibility purposes. with the understanding that the CR type could change between now and whatever we decide to call v1, i thought it made sense to version our apis and start w/ alpha.

that said, we'll have to duplicate a lot code within each api version to pull it off.

Reply:
My view is that this upgrade mechanism will only be useful once we have a version that's in use. If we release this as tech preview at some point, possibly before OCP 4.0, then maybe that should be called something like this (though still not alpha, I'm guessing). Otherwise and at this stage, I don't see it being in use an requiring version upgrades.

Reply:
done